eng/Directory.Build.Data.targets

```diff
@@ -33,12 +35,20 @@
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-004" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(SignAssembly)' != 'false'">
+  <!-- If signing is enabled for an SDK library, delay sign using the shared key -->
+  <PropertyGroup Condition="'$(SignAssembly)' != 'false' and '$(IsTestProject)' != 'true' ">
     <SignAssembly>true</SignAssembly>
-    <DelaySign>true</DelaySign>
+    <DelaySign >true</DelaySign>
```

Comment:
Looks like a extra space was added to the opening tag I'm surprised that even parses as valid xml.

Reply:
Thanks!  Good catch.